arwen/contexts/async.go

```diff
@@ -231,11 +246,24 @@ func (context *asyncContext) isValidCallbackName(callback string) bool {
 		return false
 	}
 
+	err := context.host.Runtime().ValidateCallbackName(callback)
+	if err != nil {
+		return false
+	}
+
+	return true
+}
+
+func (context *asyncContext) callExists(destination []byte) bool {
+	_, _, err := context.findCall(destination)
+	if errors.Is(err, arwen.ErrAsyncCallNotFound) {
```

Comment:
in case of other errors ?

Reply:
`findCall` does not return any other error.

Simplified with `if err != nil { return false }`, though, because it's equivalent.